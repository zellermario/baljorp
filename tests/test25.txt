create SJ kezdes 0
create SJ vegzes 0
addNeighbor kezdes 1 vegzes
addNeighbor vegzes 2 kezdes

create Jegesmedve maci kezdes

move maci 1
pass maci