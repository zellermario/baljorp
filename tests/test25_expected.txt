Fields:
SJ kezdes (0) - objects: x
	neighbours: [(1, vegzes)]
SJ vegzes (0) - objects: x
	neighbours: [(2, kezdes)]

Buildings:

Actors:
> Medve maci on vegzes
